node/service/src/chain_spec/test_spec.rs

```diff
@@ -104,7 +104,7 @@ pub fn lazy_loading_spec_builder(para_id: ParaId) -> sc_chain_spec::ChainSpecBui
 		)
 		.expect("Provided valid json map"),
 	)
-	.with_genesis_config(crate::chain_spec::moonbeam::testnet_genesis(
+	.with_genesis_config(moonbeam_runtime::genesis_config_preset::testnet_genesis(
```

Comment:
```suggestion
	.with_genesis_config_preset_name(sp_genesis_builder::DEV_RUNTIME_PRESET)
```

Reply:
Addressed in https://github.com/moonbeam-foundation/moonbeam/pull/3138/commits/6ef78e33b5f89ecdb53d07da071a3f7de6f1091a